images/machine/openipmi/ipmisim1.emu

```diff
@@ -0,0 +1,39 @@
+# This is an example simulation setup for ipmi_sim.  It creates a single
+# management controller as a BMC.  That will have the standard watchdog
+# sensor and we add a temperature sensor.
+
+# The BMC is the MC at address 20
+mc_setbmc 0x20
+
+# Now add the BMC
+mc_add 0x20 0 no-device-sdrs 0x23 9 8 0x9f 0x1291 0xf02 persist_sdr
+sel_enable 0x20 1000 0x0a
+
+# Watchdog sensor.  This must be sensor zero.
+sensor_add 0x20 0 0 35 0x6f event-only
+sensor_set_event_support 0x20 0 0 enable scanning per-state \
+	000000000001111 000000000000000 \
+	000000000001111 000000000000000
+
+# Add a temperature sensor and its SDR.  Note that sensor 0 is already
+# taken as the watchdog sensor.
+sensor_add 0x20 0 1 0x01 0x01
+# Start with the value set to 0x60
+sensor_set_value 0x20 0 1 0x60 0
+# Set just the upper thresholds with the values 0x70, 0x90, and 0xa0
+sensor_set_threshold 0x20 0 1 settable 111000 0xa0 0x90 0x70 00 00 00
+# Enable all upper threshold events events
+sensor_set_event_support 0x20 0 1 enable scanning per-state \
+	000111111000000 000111111000000 \
+	000111111000000 000111111000000
+
+# Add a memory module presence sensor.
+sensor_add 0x20 0 2 37 0x6f
+sensor_set_bit_clr_rest 0x20 0 2 1 1
+#sensor_set_bit_clr_rest 0x20 0 2 0 1
+sensor_set_event_support 0x20 0 2 enable scanning per-state \
+	000000000000011 000000000000011 \
+	000000000000011 000000000000011
+
+# Turn on the BMC
```

Comment:
I was able to add some FRU data, too. I can provide it in a follow-up if you are interested. :)

```
❯ docker exec -it test ipmitool -C 3 -I lanplus -U ADMIN -P ADMIN -H 127.0.0.1 fru
FRU Device Description : Builtin FRU Device (ID 0)
 Chassis Type          : Other
 Chassis Part Number   : MINI-LAB-C12345
 Chassis Serial        : MINI-LAB-CHASSIS-1
 Board Mfg Date        : Wed May 14 00:00:00 2025 UTC
 Board Mfg             : metal-stack
 Board Product         : NONE
 Board Serial          : B1-1234567890
 Board Part Number     : MINI-LAB-B1
 Product Manufacturer  : metal-stack
 Product Name          : NONE
 Product Part Number   : MINI-LAB-P1
 Product Version       : NONE
 Product Serial        : MINI-LAB-12345

Reply:
Yes, please add it into a separate PR.